packages/cf-component-dropdown/src/Dropdown.js

```diff
@@ -81,4 +128,4 @@ Dropdown.childContextTypes = {
   dropdownRegistry: PropTypes.instanceOf(DropdownRegistry).isRequired
 };
 
-export default Dropdown;
+export default createComponent(styles, Dropdown, ['align']);
```

Comment:
Why do we explicitly allow `align` here? Doesn't `createComponent` read from `propTypes`?